python/ray/util/client/server/dataservicer.py

```diff
@@ -18,6 +18,7 @@
 class DataServicer(ray_client_pb2_grpc.RayletDataStreamerServicer):
     def __init__(self, basic_service: "RayletServicer"):
         self.basic_service = basic_service
+        basic_service.data_servicer = self
```

Comment:
yuck cyclic reference :(

Reply:
Let me do the right thing here, I think I can make that better-ish